packages/hub/src/app/about/page.tsx

```diff
@@ -0,0 +1,58 @@
+import ReactMarkdown from "react-markdown";
+import { NarrowPageLayout } from "@/components/layout/NarrowPageLayout";
+import remarkGfm from "remark-gfm";
+import {
+  GITHUB_URL,
+  GITHUB_DISCUSSION_URL,
+  DISCORD_URL,
+  GRAPHQL_URL,
+  NEWSLETTER_URL,
+  QURI_DONATE_URL,
+} from "@/lib/common";
+
+const markdown = `
+# About Squiggle Hub
+Squiggle Hub serves as a communal platform for the creation and sharing of code written in [Squiggle](https://www.squiggle-language.com/). Both the platform and the source code are open-source and free to use.
```

Comment:
> communal platform

Is this a common way to spell this? (compared to "a community platform").

(Also tbh I'd remove the "community" word until we get comments and other social features.)

Reply:
> open-source

Link to GitHub here?